main.tf

```diff
@@ -29,8 +29,9 @@ resource "kubernetes_secret" "lacework_config" {
 
 resource "kubernetes_daemonset" "lacework_datacollector" {
   metadata {
-    name      = var.lacework_agent_name
-    namespace = var.namespace
+    name                   = var.lacework_agent_name
+    namespace              = var.namespace
+    revision_history_limit = var.revision_history_limit
```

Comment:
I think this attribute belongs to the `spec` block below, besides the `selector` block

<img width="801" alt="Screen Shot 2021-08-09 at 8 25 37 AM" src="https://user-images.githubusercontent.com/5712253/128713723-767959d5-d1a7-4fe5-9dd1-e407929af2e4.png">

https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/daemonset#spec

Reply:
done, moved it under spec